pkg/policy/api/egress_test.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !privileged_tests
+
+package api
+
+import (
+	"fmt"
+	"net"
+
+	. "gopkg.in/check.v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (s *PolicyAPITestSuite) TestHasDerivativeRuleWithoutToGroups(c *C) {
```

Comment:
Change the test name to `TestRequiresDerivative` and run multiple tests in it. This function and the next one should be joined.

Reply:
I do not agree with this one :-(. 

One test should test only one thing. If we have a test that is TestHasDerivative should test only *one* thing.

Larger test means that should have logic, lest factories, builders, etc.. and test description is not what the test have.